src/main/java/com/softwire/todos/JiraIssueReference.java

```diff
@@ -0,0 +1,21 @@
+package com.softwire.todos;
+
+import com.atlassian.jira.rest.client.api.domain.Issue;
+
+public class JiraIssueReference {
+    private final String id;
+    private final Issue issue;
+
+    public JiraIssueReference(String id, Issue issue) {
+        this.id = id;
+        this.issue = issue;
+    }
+
+    public Issue getIssue() {
```

Comment:
please could you mark this with a "@Nullable" annotation? I think conventionally people assume properties like this are not-null